django/core/handlers/asgi.py

```diff
@@ -327,7 +327,7 @@ async def send_response(self, response, send):
             # - Consume via `__aiter__` and not `streaming_content` directly, to
             #   allow mapping of a sync iterator.
             # - Use aclosing() when consuming aiter.
-            #   See https://github.com/python/cpython/commit/6e8dcda
+            #   See https://github.com/python/cpython/commit/6e8dcdaaa49d4313bf9fab9f9923ca5828fbb10e  # noqa: E501
```

Comment:
Is it not enough to add one `a`?
```suggestion
            #   See https://github.com/python/cpython/commit/6e8dcdaa
```

Reply:
Maybe, until it breaks again, either when the repository gets bigger or when GitHub decides to fix [this vulnerability](https://trufflesecurity.com/blog/anyone-can-access-deleted-and-private-repo-data-github)…it seems safer to use the URLs that GitHub provides than ones we’ve manually constructed.